avr-specs/sync-from-upstream.py

```diff
@@ -28,6 +28,12 @@
     "attiny88": {
         "cpu": "attiny88",
     },
+    "attiny841": {
+        "cpu": "attiny841",
```

Comment:
I think this needs to be `avr25` to recreate your original spec files:

```suggestion
        "cpu": "avr25",
```